### DISCUSSION TITLE
Node name update when importing renamed group address

### DISCUSSION THREAD

**Discussion by ch-zacmo** [Supergiovane/node-red-contrib-knx-ultimate#410] [Q&A]
Hi,

With my work flow, it happen a lot of time that I'm renaming group address in my project. I then make a new export of the GA for node-red. Sometimes I go on each node and refresh the name, and sometime I just let it like this because I don't have time for that.

Would it be possible to have a function or anything else that can update the node name on the flow ?

I can imagine that it is maybe more complicated than it sound because of the way of how node-red store all the node data and the API behind it.

If this is not directly possible inside knx ultimate, I will probably develop a tool to do it and share it if some have interest in it.

Maybe my workflow just sucks :)

**Comment by Supergiovane** [comment]
Hi!
Thanks for using my node.
Node-Red doesn’t unfortunately allow changing the name of flow’s nodes. 
I also have your problem ;-)
But yours is a good idea. I’ll ditch in the Node-Red API to search something useful, but i’m not sure if it allows to do that.
